<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlazeLoaderDialogBase</class>
 <widget class="QDialog" name="BlazeLoaderDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blaze Map Loader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="folderLayout">
     <item>
      <widget class="QLabel" name="folderLabel">
       <property name="text">
        <string>Blaze Output Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="folderLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="downloadTopoCheckBox">
     <property name="text">
      <string>Download NSW topographic layers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="gpkgLayout">
     <item>
      <widget class="QLabel" name="gpkgLabel">
       <property name="text">
        <string>NSW Topo GPKG Save Location (optional):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="gpkgLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="gpkgBrowseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="addMagNorthCheckBox">
     <property name="text">
      <string>Add magnetic north lines</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="zoomToExtentCheckBox">
     <property name="text">
      <string>Zoom to layer extent</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="addControlsCheckBox">
     <property name="text">
      <string>Add controls layer</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
